.github/workflows/ci.yml

```
@@ -17,6 +17,10 @@ env:
 permissions:
   contents: read
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
```

```suggestion
  group: ci-${{ github.head_ref || github.run_id }}
```
I used to use `github.workflow` until we ran into issues of people changing the title or file to "clean up" things and messing the concurrency up.